src/metrics/metrics.md

```diff
@@ -1,11 +1,62 @@
 # Metrics
 
-Here we provide definitions and information about our standard metrics.
+Here we provide definitions and information about our standard metrics.  We use some terminology that is defined [here](https://docs.telemetry.mozilla.org/concepts/terminology.html).
 
-Table of contents with links to all metrics.
+## Table of Contents
+
+<!-- toc -->
+
+-----
+
+## Daily Active Users (DAU)
+
+### Overview
+
+Daily Active Users or DAU counts the number of unique profiles active in the product on each day.  This is intended to approximate the number of people using the product each day.
+
+### Details
+
+DAU counts unique profiles. Keep in mind that a [profile is not necessarily a user](/concepts/analysis_gotchas.html#profiles-vs-users)
+
+The concept of being "active" is defined as in the [`usage criterion`](./usage.md) section and will vary by product.  A product may even have multiple possible `usage criteria`, but there will be one default choice that is our best approximation of general product usage.
+
+### Caveats
+
+If the number of users stays constant, but the average number of active profiles per user increases, this metric will tend to increase.  For more details on the relationship between users and profiles, see [the profiles vs users section in analysis gotchas](https://docs.telemetry.mozilla.org/concepts/analysis_gotchas.html#profiles-vs-users).
+
+### Dashboards
+
+This metric is available on the [standard Growth and Usage Dashboard (GUD)](https://go.corp.mozilla.com/gud) for most products and with some slicing available.
+
+### Tables
+
+DAU can easily be calculated from the Smoot tables; for example:
```

Comment:
Where's the reference documentation for these?

Reply:
@jklukas do we have anything we can point to here?

Reply:
We do not have reference documentation for these, and I do not consider the format of these tables stable enough to feel comfortable recommending that data users rely on them. The closest we have to reference documentation is these two STMO queries:

https://sql.telemetry.mozilla.org/queries/65338/source
https://sql.telemetry.mozilla.org/queries/65337/source

We have existing documentation in DTMO for DAU queries on top of the exact_mau28 tables and on top of clients_daily which are stable interfaces. I hope in H2 to refactor smoot tables and better codify the mapping of backend fields to logical application names and usage criteria, but there is currently risk in hitting the smoot tables.

Reply:
OK.  I'll use exact_mau28 instead and add the link.  Seems like it's better to stick with a standard and as high-level as possible of a table.  We may want to consider adding additional information for lower level queries at some point (like how to calculate MAU from raw data).   But perhaps that can wait.

Reply:
Thank you both!